js/script.js

```diff
@@ -0,0 +1,104 @@
+var counter = 0;
+var intervalId;
+
+$(document).ready(function() {
+
+	$('#reddit-search').on('click', function(e) {
+	    e.preventDefault();
+	    search($('#input').val());
+	    console.log('clicked'+$('#input').val())
+	    $('#input').val('');
+  	})
+
+  		$('#stop-search').on('click', function(e) {
+			    e.preventDefault();
+			    $('#images').html('');
+
+			    clearInterval(intervalId);
+
+  			})
+
+	function search(input) {
+		$.get('http://www.reddit.com/search.json', {q:input},"+nsfw:no").done(function(data) {
```

Comment:
This isn't actually adding nsfw:no to your query. Instead of specifying nsfw:no as an additional parameter, concatenate it with the input string.